src/main/java/org/jenkinsci/plugins/workflow/cps/FlowHead.java

```diff
@@ -94,13 +94,15 @@ public CpsFlowExecution getExecution() {
     }
 
     void newStartNode(FlowStartNode n) throws IOException {
-        for (Action a : execution.flowStartNodeActions) {
```

Comment:
[diff for review](https://github.com/jenkinsci/workflow-cps-plugin/pull/158/files?w=1)